theano/gof/tests/test_vm.py

```diff
@@ -5,6 +5,8 @@
 
 from nose.plugins.skip import SkipTest
 import numpy
+import theano
+
```

Comment:
pep8 tell that we should have 3 blocs of import:
1) python related import
2) 3th party module import
3) this project import.

Move this import at line 15?